### ISSUE TITLE
can't find IDMap.json after calling setPath("userData") 

### ISSUE THREAD

**Issue by chbdetta** [MarshallOfSound/electron-devtools-installer#89]
After modifying the userData path, installExtension will throw:
```
Error: ENOENT: no such file or directory, open '/Users/chenhuanbin/Library/Application Support/Anki Editor/extensions/IDMap.json'
```

**Comment by chbdetta** [comment]
I've submit a quick fix pr: #88

**Comment by MarshallOfSound** [comment]
Fixed in #88
